### ISSUE TITLE
Anyone experiencing Out of Memory issues when using this with Kinesis?

### ISSUE THREAD

**Issue by timogoosen** [guardian/kinesis-logback-appender#13]
Is anyone experiencing Out Of Memory issues when using this with Kinesis?

**Comment by sebastiankirsch** [comment]
Yes, we do. But only for a simple setup (i.e. a tiny application that only processes SQS messages); more complex applications don't exhibit the behavior.
In our case, it seems like the old generation is not gc'ed. Scheduling a call to `System.gc()` did the trick for us.

We (https://github.com/ImmobilienScout24) have an AWS support case open to investigate this.
